tests/integration/cmd_push_test.go

```diff
@@ -37,6 +38,27 @@ var _ = Describe("odo push command tests", func() {
 		os.Unsetenv("GLOBALODOCONFIG")
 	})
 
+	Context("Check for label propagation after pushing", func() {
+
+		FIt("Check for labels", func() {
+			helper.CmdShouldPass("git", "clone", "https://github.com/openshift/nodejs-ex", context+"/nodejs-ex")
+			helper.CmdShouldPass("odo", "component", "create", "nodejs", cmpName, "--project", project, "--context", context+"/nodejs-ex", "--app", appName)
+			helper.CmdShouldPass("odo", "push", "--context", context+"/nodejs-ex")
+
+			// Always compiled with a different version... this is possibly the ONLY portion of tests
+			// where we import from the main odo "pkg" code. We need this to make sure the version is correctly
+			// propagated.
```

Comment:
Then we are not verifying the version, rather just displaying what is there stored in VERSION variable of package ```github.com/openshift/odo/pkg/version```. For example, each time the test will pass even you use ```VERSION = ""```. Use regular expression instead of importing version the package. 

See for reference - https://github.com/openshift/odo/blob/master/tests/integration/generic_test.go#L273

Reply:
Use it
```
versions := helper.CmdShouldPass("odo", "version")
re := regexp.MustCompile(`odo\s+v.\S*`)
odoVersion := re.FindString(versions)
```